src/components/navbar-editor.tsx

```diff
@@ -247,11 +247,12 @@ export default function EditorNavbar(props: EditorNavbarProps) {
 					props.roomState?.value.participants.filter((participant) => {
 						if(participant.isHost) return true
 						return false
-					})[0]?.userEmail === props.persistenceState.value.session?.user.email ? props.persistenceState.value.session?.user.email : "???"
+					})[0]?.userEmail === props.persistenceState.value.session?.user.email ? props.persistenceState.value.session?.user.email : "the host"
```

Comment:
can this string literal be made into a constant and given a good name, like "defaultPersistenceStateEmail" or something?